src/index.js

```diff
@@ -1,6 +1,22 @@
 import './style.css';
+import Board from './board.js';
 import leaderBoard from './leaderBoard.js';
 
 const board = leaderBoard();
 const root = document.getElementById('root');
 root.appendChild(board);
+
+const game = new Board();
+
+async function startGame() {
```

Comment:
- [ ] Please, don't use function word, in place, use the arrow function as ES6 good practices say, this will help to get a better project on this helpful practices. 🐶